packages/ui/src/memberships/queries/members.graphql

```diff
@@ -23,6 +23,34 @@ query GetMembers($rootAccount_in: [String!], $controllerAccount_in: [String!], $
   }
 }
 
+query FilterMembers(
+  $id: ID
+  $search: String
+  $isVerified: Boolean
+  $isFoundingMember: Boolean
+  # $isCouncil: Boolean!
+  # role: ???
```

Comment:
You can safely omit comments like this. If there's a TODO a better approach would be to create an issue on GH with things that were left to do. However, if this is a part of a ticket that you're working on, you don't have to create a separate one.

In this case, a follow-up is welcomed as we have to remember that in the feature when we get the `isCouncil` nad roles from the query node.

Reply:
I removed the comments and created #525